WebGL/CUDLR_Relay.jslib

```diff
@@ -0,0 +1,37 @@
+var CUDLR_Relay = {
+  $cr_callback: function(){},
+
+  SetCudlrCallbackAndCreateInput: function(callbackPtr){
+    cr_callback = callbackPtr;
+    
+    window.cc = function(command) {
```

Comment:
This method invoke `CUDLR.Console.Run()` and output result on browser console.
You can also type like `cc("object list")` in the broser console command line to Run it.